modules/builds-troubleshooting-service-certificate-generation.adoc

```diff
@@ -8,24 +8,29 @@
 If your request for access to resources is denied:
 
 Issue::
-If a service certificate generation fails with (service's
-`service.alpha.openshift.io/serving-cert-generation-error` annotation
-contains):
+If a service certificate generation fails with (service's `service.alpha.openshift.io/serving-cert-generation-error` annotation contains):
 
+.Example output
+[source,terminal]
 ----
 secret/ssl-key references serviceUID 62ad25ca-d703-11e6-9d6f-0e9c0057b608, which does not match 77b6dd80-d716-11e6-9d6f-0e9c0057b60
 ----
 
 Resolution::
-The service that generated the certificate no longer exists, or has a different
-`serviceUID`. You must force certificates regeneration by removing the old
-secret, and clearing the following annotations on the service
-`service.alpha.openshift.io/serving-cert-generation-error`,
-`service.alpha.openshift.io/serving-cert-generation-error-num`:
+The service that generated the certificate no longer exists, or has a different `serviceUID`. You must force certificates regeneration by removing the old secret, and clearing the following annotations on the service `service.alpha.openshift.io/serving-cert-generation-error`, `service.alpha.openshift.io/serving-cert-generation-error-num`:
```

Comment:
This sentence doesn't make a whole lotta sense to me!

Reply:
Fair, edited.